pagopa/io-app
# feat: [PE-551] Category detail screen

## Short description
This PR implements the new DS on CGN category detail screen.

## List of changes proposed in this pull request
- Adds the new design elements on `CgnMerchantsListByCategory`

> [!note]
> Categories list and all merchants list has been refactored on #5668

## How to test
Navigate to CGN categories and check the detail of a single category.


https://github.com/pagopa/io-app/assets/3959405/3f7e6d42-77e1-4d8d-9325-0917c55a25ad
